### PR TITLE
Accept empty string as Path Params + extra

### PR DESCRIPTION
**Pull request by Zehir**
Fix for #402
It's will now correctly find errors when the path have empty params.

Add missing dev dependencies.
Add gitignore for .devcontainer as many users like me use it to dev.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -125,7 +125,7 @@ export function findEndpointErrorsByAlias(
 }
 
 export function pathMatchesUrl(path: string, url: string) {
-  return new RegExp(`^${path.replace(paramsRegExp, () => "([^/]+)")}$`).test(
+  return new RegExp(`^${path.replace(paramsRegExp, () => "([^/]*?)")}$`).test(
```
Comment (ecyrbe) [inline src/utils.ts:128]: `*?` is not necessary, `*` is enough.
```suggestion
  return new RegExp(`^${path.replace(paramsRegExp, () => "([^/]*)")}$`).test(
```

**Review by ecyrbe — CHANGES_REQUESTED** [review]
@Zehir thank you for the PR.

Please remove the pnpm-lock.yaml, the v10 version is still using yarn. only v11 will use pnpm.

**Comment by Zehir** [comment]
I made the requested updates

File: package.json [hunk 2]
```diff
@@ -54,12 +54,14 @@
     "zod": "^3.x"
   },
   "devDependencies": {
+    "@jest/types": "^29.5.0",
```
Comment (Zehir) [inline package.json:57]: @ecyrbe For thoses version you want exact version or you accept later versions ?

Comment (ecyrbe) [reply package.json:57]: Oh, yes exact versions. To have precise control on the updates. I use renovate bot to handle updates for me. It allows to detect breaking changes in dependencies with a PR.

**Comment by Zehir** [comment]
updated

**Pull request merged**